### PR TITLE
fixed remaining problems

### PR DESCRIPTION
**Pull request by RationalityFrontline**
The remaining problem is irrelevant to jsemver, but related to mockito and a false unit test SingletonExtensionFactoryTest.

### CODE REVIEW

File: pf4j/src/test/java/org/pf4j/SingletonExtensionFactoryTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void createNewEachTimeFromDifferentClassLoaders() throws Exception {
     private URL[] getClasspathReferences() throws MalformedURLException {
         String classpathProperty = System.getProperty("java.class.path");
 
-        String[] classpaths = classpathProperty.split(":");
+        String[] classpaths = classpathProperty.split(";");
```
Comment (decebals) [inline pf4j/src/test/java/org/pf4j/SingletonExtensionFactoryTest.java:77]: I don't know what to say here. Probably the correct approach is to use `File.pathSeparator`.
The `;` character is used to separate the file paths on Windows.

File: pom.xml [hunk 2]
```diff
@@ -51,7 +51,7 @@
 
         <junit.version>5.4.0</junit.version>
         <hamcrest.version>2.1</hamcrest.version>
-        <mockito.version>2.24.0</mockito.version>
+        <mockito.version>3.8.0</mockito.version>
```
Comment (decebals) [inline pom.xml:54]: I prefer to put this change in a separate PR.

**Review by decebals — COMMENTED** [review]
Also, the title pf PR is very general:
```
fixed remaining problems
```
Is it related to other PR?

**Comment by RationalityFrontline** [comment]
Sorry for the mess. In order to make things clear, I created two separate pr with corresponding issues: #435, #436.
